src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceSendLiveAccountCreatedTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.pay.adminusers.resources;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.pay.adminusers.model.Service;
+
+import static java.lang.String.format;
+import static org.hamcrest.core.Is.is;
+import static uk.gov.pay.adminusers.fixtures.GovUkPayAgreementDbFixture.govUkPayAgreementDbFixture;
+import static uk.gov.pay.adminusers.fixtures.ServiceDbFixture.serviceDbFixture;
+
+public class ServiceResourceSendLiveAccountCreatedTest extends IntegrationTest {
```

Comment:
I think `Email` wants to be in here somewhere

Reply:
🤦‍♂️ yes it probably does..